frontend/src/game/scenes/Collect.ts

```diff
@@ -34,6 +45,7 @@ export class Collect extends Scene {
     EventBus.on("upgrade-purchased", (upgrades) => {
       const bci = calculateVariableValue(upgrades, GameVariable.BitCheckInterval);
       const bap = calculateVariableValue(upgrades, GameVariable.BitAppearanceProbability);
+      const sweepCheck = calculateVariableValue(upgrades, GameVariable.BitSweeperSize);
```

Comment:
unused variable

Reply:
wow ok thanks. nope ill just delete it and remake it entirely until its to your liking dictator joseph.